src/components/MapView/Layers/PointDataLayer/index.tsx

```diff
@@ -28,54 +26,58 @@ function PointDataLayer({ layer }: { layer: PointDataLayerProps }) {
   const dispatch = useDispatch();
 
   const { data } = layerData || {};
+  const { features } = data || {};
   const { t } = useSafeTranslation();
   useEffect(() => {
-    if (!data) {
+    if (!features) {
```

Comment:
why the switch to testing on features vs data?

Reply:
because i changed the type of `PointLayerData` to `{features: PointData[] }`, and when `AdminLayerDisplayed` option is on, it will return type `AdminDataLayerData`. Both these types store the location points in `features`, hence we are checking for features instead of data